torchci/components/JobSummary.tsx

```diff
@@ -6,6 +6,7 @@ export default function JobSummary({ job }: { job: JobData }) {
     <>
       <JobConclusion conclusion={job.conclusion} />
       <a href={job.htmlUrl}> {job.jobName} </a>
+      <b> {job.branch} </b>
```

Comment:
> Can we only bold the branch name if it's the "master" branch?

I think there was some misunderstanding with my [original comment](https://github.com/pytorch/test-infra/pull/969#discussion_r1008241570).

I meant that when `job.branchname == "master"`, the html here should look like:
`<b> [{job.branch}] </b>` (how it originally looked)

While at all other times it should effectively look like:
`<span> [{job.branch}] </span>` (or something like that